Subject: Add docs related to CMake

These are only quick README adjustments and fixes on how to use CMake
with PHP.
---
 README.md | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/README.md b/README.md
index c5376f6772..c39945251d 100644
--- a/README.md
+++ b/README.md
@@ -35,8 +35,35 @@ For other systems, see the [installation chapter](https://www.php.net/install).
 
 ### Building PHP source code
 
+#### CMake
+
+On Debian-based systems:
+
+    sudo apt install -y cmake gcc g++ bison re2c libxml2-dev libsqlite3-dev
+
+On Fedora-based systems:
+
+    sudo dnf install cmake gcc gcc-c++ bison re2c libxml2-devel sqlite-devel
+
+Create a build directory:
+
+    mkdir php-build
+
+Generate build system:
+
+    cd php-build
+    cmake ..
+
+Build PHP:
+
+    cmake --build .. -j
+
+#### Windows using JScript
+
 *For Windows, see [Build your own PHP on Windows](https://wiki.php.net/internals/windows/stepbystepbuild_sdk_2).*
 
+#### Autotools
+
 For a minimal PHP build from Git, you will need autoconf, bison, and re2c. For
 a default build, you will additionally need libxml2 and libsqlite3.
 
